app/questionnaire/questionnaire_schema.py

```diff
@@ -215,20 +216,52 @@ def get_post_submission(self) -> ImmutableDict:
         schema: ImmutableDict = self.json.get("post_submission", ImmutableDict({}))
         return schema
 
+    def _is_list_name_in_rule(
+        self, when_rules: Union[dict, Sequence], list_name: str
```

Comment:
Should probably be just rule

Reply:
https://github.com/ONSdigital/eq-questionnaire-runner/pull/729#discussion_r768867516